index.html

```diff
@@ -58,7 +60,221 @@ <h3>
 
 </section>
    
+
+<header class="title-header">
+    <h2 class="title">My Recent Works</h2>
+  </header>
+  <section class="recent-works">
+      <div class="work">
+          <section >
+  <img src="/img/Img-Placeholder.png" alt="">
+          </section>
+          <div class="mg">
+  
+        
+          <section class="stories">
+             Multi-Post Stories
+          </section>
```

Comment:
- [ ] This should be a header kindly use one of the `<hx>` tags.
- [ ] Kindly fix the same issue in all the other projects